### PR TITLE
Run eslint --fix

### PR DESCRIPTION
**Pull request by janicduplessis**
CI is currently failing because of a lint issue, this fixes it and a bunch of other warnings that are auto-fixable.

**Test plan**
Quick manual test, cosmetic changes only.

### CODE REVIEW

**Comment by janicduplessis** [comment]
CI still fails but at least one step further :)

**Comment by fernandopasik** [comment]
One step closer to getting done https://github.com/facebook/react-native/issues/15726

**Comment by janicduplessis** [comment]
@TheSavior Could you ship this, looks like you were the last one to play with lint stuff :)

**Comment by grabbou** [comment]
I have submitted #16259 which is slightly smaller than this PR and needed it to release new RC

**Comment by grabbou** [comment]
Wish I saw this PR earlier before struggling to understand why my build has
failed. Thanks Janic!

File: local-cli/link/ios/unlinkAssets.js [hunk 1]
```diff
@@ -19,14 +19,14 @@ module.exports = function unlinkAssetsIOS(files, projectConfig) {
   if (!plist) {
     return log.error(
       'ERRPLIST',
-      `Could not locate Info.plist file. Check if your project has 'INFOPLIST_FILE' set properly`
+      'Could not locate Info.plist file. Check if your project has \'INFOPLIST_FILE\' set properly'
```
Comment (TheSavior) [inline local-cli/link/ios/unlinkAssets.js]: These are a bummer

**Comment by TheSavior** [comment]
This failed to import. We are working on getting the import fixed

**Comment by TheSavior** [comment]
Looks like this landed. Thanks for contributing!

**Pull request closed without merging**